src/main/resources/com/mathworks/ci/pipeline/RunMatlabTestsStep/help-testResultsHTML.html

```diff
@@ -0,0 +1,12 @@
+<div>
+<br>
```

Comment:
If multiple copies of the same help text must be maintained, please apply my earlier comments globally (i.e., multiple times).